tests/nnstreamer_flatbuf/runTest.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+##
+## SPDX-License-Identifier: LGPL-2.1-only
+##
+## @file runTest.sh
+## @author Gichan Jang <gichan2.jang@samsung.com>
+## @date May 25 2020
+## @brief SSAT Test Cases for flatbuffers subplugin of tensor converter and decoder
+## @details After decoding the tensor into flatbuffer, convert it to tensor(s) again to check if it matches the original
+##
+if [[ "$SSATAPILOADED" != "1" ]]; then
+    SILENT=0
+    INDEPENDENT=1
+    search="ssat-api.sh"
+    source $search
+    printf "${Blue}Independent Mode${NC}
+"
+fi
+
+# This is compatible with SSAT (https://github.com/myungjoo/SSAT)
+testInit $1
+
+if [ "$SKIPGEN" == "YES" ]; then
+    echo "Test Case Generation Skipped"
+    sopath=$2
+else
+    echo "Test Case Generation Started"
+    python ../nnstreamer_converter/generateGoldenTestResult.py
+    python ../nnstreamer_merge/generateTest.py
+    sopath=$1
+fi
+convertBMP2PNG
+
+PATH_TO_PLUGIN="../../build"
+
+gst-launch-1.0 -m -v --gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=1 ! video/x-raw,format=RGB,width=280,height=40,framerate=0/1 ! tee name=t ! queue ! videoconvert ! video/x-raw, format=BGRx ! tensor_converter silent=TRUE ! filesink location="test.bgrx.log" sync=true t. ! queue ! filesink location="test.rgb.log" sync=true
+
+gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=1 ! video/x-raw,format=RGB,width=280,height=40,framerate=0/1 ! tee name=t ! queue ! videoconvert ! video/x-raw, format=BGRx ! tensor_converter silent=TRUE ! tensor_decoder mode=flatbuf ! other/flatbuf-tensor ! tensor_converter ! filesink location=\"test.bgrx.log\" sync=true t. ! queue ! filesink location=\"test.rgb.log\" sync=true" 1R 0 0 $PERFORMANCE
+
+gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=1 ! video/x-raw,format=GRAY8,width=280,height=40,framerate=0/1 ! queue ! tensor_converter silent=TRUE ! tensor_decoder mode=flatbuf ! other/flatbuf-tensor ! tensor_converter ! filesink location=\"test.gray8.log\" sync=true" 1G 0 0 $PERFORMANCE
+
+callCompareTest testcase01.bgrx.golden test.bgrx.log 1-1 "BGRX Golden Test" 1 0
+callCompareTest testcase01.rgb.golden test.rgb.log 1-2 "RGB Golden Test" 1 0
+callCompareTest testcase01.gray8.golden test.gray8.log 1-3 "Gray8 Golden Test" 1 0
```

Comment:
videotestsrc includes randomely generated pixels. How do you create proper testcase01.*.golden?

Reply:
You're right.. It's weird how it passed. Maybe the first frame of videotestsrc is consistent?
I will change videotestsrc to filesrc and related other tests (tensor converter etc.)

Reply:
IMO, `videotestsrc` generates deterministic random values, and there're some testcases using `videotestsrc` and its golden data.

Reply:
> IMO, `videotestsrc` generates deterministic random values, and there're some testcases using `videotestsrc` and its golden data.

Right, this is the case: https://github.com/nnstreamer/nnstreamer/blob/49fa3a91ef5749ae834c079a2d66e4fa78a46510/tests/nnstreamer_decoder_image_segment/runTest.sh#L77-L79

Reply:
> IMO, `videotestsrc` generates deterministic random values, and there're some testcases using `videotestsrc` and its golden data.

Yes, `videotestsrc` starts with a fixed seed of 0 and update as the function below (all deterministic it seems)
https://github.com/GStreamer/gst-plugins-base/blob/648acbe726a74b547d6a4003265e492cefe69a9f/gst/videotestsrc/videotestsrc.c#L38-L42

Reply:
Let's not assume that videotestsrc's randomly generated pixels will always be exactly repeated with every restart.

Refer to https://gstreamer.freedesktop.org/documentation/videotestsrc/index.html?gi-language=c , use "pattern" parameter to set some "complex pattern" without random pixels.

Reply:
I removed the randomly generated patterns and changed to pattern no.13.

![Screenshot from 2020-07-01 13-38-25](https://user-images.githubusercontent.com/56856496/86203629-74629280-bba0-11ea-918d-d9131fce13f5.png)